### PR TITLE
Suggested search page

### PR DESCRIPTION
**Pull request by defong**
image

image

image

image

### CODE REVIEW

**Comment by fofr** [comment]
Should we have some tests for this behaviour?

File: src/Views/Results/Components/SuggestedSearch/Default.cshtml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <li>
+          <a href="@(Url.Action("Index", "Results", item.ResultsFilter.ToRouteValues()))">@text</a>
+        </li>
+    }
+
+    </ul>
+}
+
+<p>
+  You can <a href="https://www.ucas.com/ucas/teacher-training/ucas-teacher-training-search-training-programmes">search for courses in other subjects</a> on the UCAS website..
```
Comment (fofr) [inline src/Views/Results/Components/SuggestedSearch/Default.cshtml]: There's two periods here. Should be 1.

File: src/Views/Results/Components/SuggestedSearch/Default.cshtml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@if(Model.Any()) {
+    <p>For business studies there are:</p>
+    <ul class="list list-bullet">
+
+    @foreach (var item in Model)
+    {
+      var courseText = " course" + (item.TotalCount == 1 ? "" : "s");
+      var areaText = item.ResultsFilter.RadiusOption.HasValue ?
+        " with " + item.ResultsFilter.GetFundingTypes().First().ToLowerInvariant() + " available within " + item.ResultsFilter.rad + " miles of " + item.ResultsFilter.loc :
```
Comment (fofr) [inline src/Views/Results/Components/SuggestedSearch/Default.cshtml]: When the financial support option being linked to is 'student finance and salaries' (ie All) can we drop the "with student finance and salaries available" bit of the text:
```diff
-9 courses with student finance and salaries available within 50 miles of Brighton, UK
+9 courses within 50 miles of Brighton, UK
```

**Comment by defong** [comment]
Fixed the text tweaks

image

image

**Comment by defong** [comment]
With regards to test.

UI layer makes multiple creates to the API to get the results, there isn't a great deal of logic for it most of it is boilerplate code.

The heaviest bit of logic is on the default.cshtml to solely display the correct text.

**Pull request closed without merging**